.github/workflows/cypress.yml

```
@@ -52,11 +52,11 @@ jobs:
         if: ${{ failure() }}
         with:
           name: cypress-video-artifacts
-          path: vets-website/**/cypress/videos
+          path: '~/_work/vets-website/vets-website/cypress/videos'
```

Does this do the same thing?

```suggestion
          path: cypress/videos
```

---

It might. I'll test it out.